src/Jackalope/Transport/DoctrineDBAL/Query/QOMWalker.php

```diff
@@ -124,7 +124,7 @@ private function getSelectorAlias($selectorName)
     }
 
     /**
-     * @param QOM\QueryObjectModelInterface $qom
+     * @param QueryObjectModel $qom
```

Comment:
Well, this is more accurate :) If were to change the signature of this method to the interface it should be a different PR.